db/range_del_aggregator.h

```diff
@@ -40,6 +40,18 @@ enum class RangeDelPositioningMode {
   kBinarySearch,
 };
 
+// TODO: think about whether this is the right thing to do here, or if having a completely different type
+// works better
+struct TruncatedRangeTombstone : public RangeTombstone {
```

Comment:
I think we'll want this to contain two `ParsedInternalKey`s to avoid copying during `AddTombstones`.